protocol/dydx-cosmwasm/src/msg.rs

```diff
@@ -108,3 +81,80 @@ impl From<DydxMsg> for CosmosMsg<DydxMsg> {
 }
 
 impl CustomMsg for DydxMsg {}
+
+#[cfg(test)]
```

Comment:
Unsure about Rust conventions, but should these unit tests to moved to separate file?

Reply:
Rust conventions keep test modules in the same file as code